src/repl/repl.ts

```diff
@@ -76,6 +76,12 @@ function validChapterVariant(chapter: any, variant: any) {
   for (const lang of sourceLanguages) {
     if (lang.chapter === chapter && lang.variant === variant) return true
   }
+  for (const lang of scmLanguages) {
+    if (lang.chapter === chapter && lang.variant === variant) return true
+  }
+  if (chapter < 0) {
+    return true
+  }
```

Comment:
I left this for easier debugging purposes. I think you can leave it out and it should improve the test coverage slightly.

Reply:
Will do, thanks!